src/mongo/MongoScrapbook.ts

```diff
@@ -224,13 +225,13 @@ class MongoScriptDocumentVisitor extends MongoVisitor<MongoCommand[]> {
 			}
 		}
 		else if (child instanceof mongoParser.ObjectLiteralContext) {
-			let propertyNameAndValue = <mongoParser.PropertyNameAndValueListContext>child.children.find((node) => node instanceof mongoParser.PropertyNameAndValueListContext);
+			let propertyNameAndValue = randomUtils.findType(child.children, mongoParser.PropertyNameAndValueListContext);
```

Comment:
randomUtils is not "random utilities", but "utilities for randomness"

Reply:
Likely need to create a new util file.

Reply:
Does it look like `utils/array.ts` would be a good place?

Reply:
yes